roles/openstack_nested_virtualization/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: copy the nested virt playbook
+  template:
+    src: nested_virt.yml.j2
+    dest: "{{ tmp_dir }}/nested_virt.yml"
+
+- name: run the playbook to enable nested virt on the hypervisors
+  shell: source "{{ ansible_user_dir }}/stackrc"; ansible-playbook -vv -i "{{ openstack_tripleo_ansible_inv }}" "{{ tmp_dir }}/nested_virt.yml"
```

Comment:
I would suggest that this format would be easier to work with, especially in the long run, but it is more personal preference:

```
- name: run the playbook to enable nested virt on the hypervisors
  shell: |
    source /home/stack/stackrc
    ansible-playbook -vv -i /usr/bin/tripleo-ansible-inventory /tmp/nested_virt.yml
```

You won't need as many quotes and the commands will be copy and paste friendly.  I would maybe even discuss/decide whether it really make sense to have all those variables, although all the previous ansible style in this repo uses many variables.  I would argue that the extra variables make the ansible less readable, less copy and paste friendly for this purpose built automation.  I don't think we would change the tmp directory, nor do I see a reason to change the netsted virt user/hosts, unless we had a specific need for such, but my thoughts are this repo is basically littered with extra variables and extra tasks that provide no additional value in this purpose built env (/me gets off soap box before he offends all authors)  I also don't know all of the past it has been though so maybe there is a need for those additional tasks/variables that hasn't been shared to me.  Just my .02 cents.